libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1896,6 +1898,74 @@ string YulUtilFunctions::copyArrayToStorageFunction(ArrayType const& _fromType,
 }
 
 
+string YulUtilFunctions::copyInlineArrayToStorageFunction(InlineArrayType const& _fromType, ArrayType const& _toType)
```

Comment:
Conceptually they might sound similar - in both cases we need to store some collection of values. The values in each collection can differ in types. But, when it comes to details I can see a lot of differences. Starting from a data source location. For structures we need to deal with calldata, memory and storage while for inline array we have only stack. For structures we need to store all the members. For inline arrays we store all members, but also need to remember about array length in case of dynamic arrays. I don't see much common parts. Do I miss something?